### PR TITLE
jdtls 1.13.0

### PR DESCRIPTION
**Pull request by Porkepix**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
Error: jdtls: failed
  An exception occurred within a child process:
    EOFError: end of file reached

**Comment by issyl0** [comment]
Looks like this [release requires Java 17](https://github.com/eclipse/eclipse.jdt.ls/issues/2117). If I change line 30 from `11` to `17` and reinstall, printing `_e.read` in the test reveals:

```
"Traceback (most recent call last):\n" +
"  File \"/opt/homebrew/Cellar/jdtls/1.13.0/libexec/bin/jdtls\", line 17, in <module>\n" +
"    jdtls.main(sys.argv[1:])\n" +
"  File \"/opt/homebrew/Cellar/jdtls/1.13.0/libexec/bin/jdtls.py\", line 75, in main\n" +
"    java_executable = get_java_executable(known_args.validate_java_version)\n" +
"  File \"/opt/homebrew/Cellar/jdtls/1.13.0/libexec/bin/jdtls.py\", line 31, in get_java_executable\n" +
"    out = subprocess.check_output([java_executable, '-version'], stderr = subprocess.STDOUT, universal_newlines=True)\n" +
"  File \"/opt/homebrew/Cellar/python@3.9/3.9.13/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py\", line 424, in check_output\n" +
"    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,\n" +
"  File \"/opt/homebrew/Cellar/python@3.9/3.9.13/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py\", line 528, in run\n" +
"    raise CalledProcessError(retcode, process.args,\n" +
"subprocess.CalledProcessError: Command '['java', '-version']' returned non-zero exit status 1.\n"
```

File: Formula/jdtls.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def install
     libexec.install "plugins"
 
     (bin/"jdtls").write_env_script libexec/"bin/jdtls",
-      Language::Java.overridable_java_home_env("11")
+      Language::Java.overridable_java_home_env
```
Comment (chenrui333) [inline Formula/jdtls.rb:30]: This was missing when updating the `openjdk`

**Comment by chenrui333** [comment]
EOFError: end of file reached

File: Formula/jdtls.rb [hunk 2]
```diff
@@ -16,8 +16,8 @@ class Jdtls < Formula
     sha256 cellar: :any_skip_relocation, all: "5af62f9cf5c42b7ba320b51901224ca87593475a80d292beeb3cbd9aede5e50f"
   end
 
-  depends_on "openjdk@11"
-  depends_on "python@3.9"
+  depends_on "openjdk"
+  depends_on "python@3.10"
```
Comment (cho-m) [inline Formula/jdtls.rb:20]: `libexec/"bin/jdtls"` uses `#!/usr/bin/env python3`, so it won't work as is with `python@3.10`. This should end up falling back to Python 3.8 on macOS, but script needs Python 3.9 at minimum.

Can either do a shebang rewrite, or a PATH override in env script.

Not sure if this is what is causing current test failure though.

**Pull request closed without merging**